### PR TITLE
language: better error message for the script runner

### PR DESCRIPTION
**Pull request by robin-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags, if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs [hunk 1]
```diff
@@ -298,7 +298,7 @@ createArchive ::
        PackageConfigFields
     -> String
     -> BSL.ByteString -- ^ DALF
-    -> [(T.Text, BS.ByteString)] -- ^ DALF dependencies
+    -> [(T.Text, BS.ByteString, LF.PackageId)] -- ^ DALF dependencies
```
Comment (cocreature) [inline compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs]: This seems to be completely unrelated to this PR.

Comment (robin-da) [reply compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs]: uh, thanks for spotting this. Seems this slipped in when I pushed after scalafmt.

Comment (cocreature) [reply compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs]: ironically scalafmt still seems unhappy :slightly_smiling_face:

**Comment by cocreature** [comment]
scalafmt is still broken :trollface:

File: compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs (hunk 1 above)
Comment (robin-da) [reply compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs]: scalafmt is trolling me ...
